src/tenderly/spawn-devnet.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
+import fs from "fs";
+import dotenv from "dotenv";
+import util from "util";
+import proc from "child_process";
+
+const asyncExec = util.promisify(proc.exec);
+
+dotenv.config();
+
+const {
+  TENDERLY_ACCESS_KEY,
+  TENDERLY_PROJECT_SLUG,
+  TENDERLY_ACCOUNT_ID,
+  TENDERLY_DEVNET_TEMPLATE,
+} = process.env;
+
+// eslint-disable-next-line max-len
+const command = `tenderly devnet spawn-rpc --project ${TENDERLY_PROJECT_SLUG} --template ${TENDERLY_DEVNET_TEMPLATE} --account ${TENDERLY_ACCOUNT_ID}  --access_key ${TENDERLY_ACCESS_KEY}`;
+
+
+const spawnDevNet = async () => {
+  const { stderr } = await asyncExec(command);
+  const devNetUrl = stderr.trim().toString();
```

Comment:
This feels super weird... we're intentionally running a command that fails, then building a variable from stderr?

Reply:
I don't think it fails necessarily. They probably just use this channel to pass the data on their side. not sure why. if it failed, it won't spawn the devnet.... or shouldn't...

Reply:
I guess it didn't logically fail, but if it's coming through stderr it's failing to the system, but if that's how they've done it you have to read it this way. Just a weird choice on their part....

Reply:
agreed